docs/UserGuide.md

```diff
@@ -72,43 +72,73 @@ Shows a message explaning how to access the help page.
 
 Format: `help`
 
+### Adding a contact: `add`
 
-### Adding a person: `add`
+Adds a contact to the address book.
 
-Adds a person to the address book.
+Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [g/GENDER] [d/DOB] [ph/NEW_PURCHASE_DETAIL] [t/TAG]`
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
+- The compulsory parameters are: name (n), phone number (p), email (e), address (a)
+- The parameters in [ ] are optional parameters, including gender (g), date of birth (d), purchase history (ph), tag (t).
```

Comment:
Perhaps format the command prefixes as inline code to signal to the reader that they are part of the command?
i.e. name (`n`), phone number (`p`), email (`e`), address (`a`)